docs/source/shared.rst

```diff
@@ -97,10 +98,11 @@ graph.
 Linear scaling with number of tasks
 ```````````````````````````````````
 
-As we increase the number of tasks in a graph we see that the scheduling
-overhead grows linearly.  The asymptotic cost per task depends on the
-scheduler.  The schedulers that depend on some sort of asynchronous pool have
-costs in the few milliseconds.  The schedulers that are single threaded are
+As we increase the number of tasks in a graph, we see that the scheduling
+overhead grows linearly.  The asymptotic (approaching a value or curve 
+arbitrarily closely) cost per task depends on the scheduler.  The schedulers 
```

Comment:
This definition of asymptotic is not particularly accurate here.  You're thinking of the mathematical term used in typical math courses.  I'm using the term used in algorithm complexity.  This section is developer focused.  I recommend removing the definition.